scipy/signal/windows/windows.py

```diff
@@ -1620,6 +1620,80 @@ def exponential(M, center=None, tau=1., sym=True):
     return _truncate(w, needs_trunc)
 
 
+def taylor(M, nbar=4, level=-30, norm=True, sym=True):
+    """
+    Return the Taylor window.
+
+    The Taylor window allows for a selectable sidelobe suppression with a
+    minimum broadening. This window is commonly used in radar processing [1]_.
+
+    Parameters
+    ----------
+    M : int
+        Number of points in the output window. If zero or less, an
+        empty array is returned.
+    nbar : int
+        Number of nearly constant level sidelobes adjacent to the mainlobe.
+    level : float
+        Desired peak sidelobe level in decibels (db) relative to the mainlobe.
```

Comment:
`level` is a negative number, but chebwin uses a positive number?  Should they use the same parameter names since one is an approximation of the other?

Reply:
I think that it makes sense to use the positive number as well, I simply left it from the original contributor's commits.

Reply:
Agreed internal consistency is better, so if `chebwin` and other functions use positive then this should, too

Reply:
What about the name? I don't see a clear agreement there as `chebwin` uses `at` for "attenuation" which to me is less clear than using `sll` for "sidelobe level".

Reply:
`at` is not a great name. I guess since it's different from `at` / `attenuation` a bit anyway. I [argued against sll before](https://github.com/scipy/scipy/pull/8032#discussion_r144728686) but I think I've come around to that name as okay, especially since that's what it's called other places.